book/schedule.yaml

```diff
@@ -45,6 +45,9 @@ tabs:
     time: 8:30 - 9:30
     leads:
     - Romina Piunno
+    resources:
+    - title: Recording
+      url: https://youtu.be/ztAAWL_dQGo
```

Comment:
Are we also listing out the tutorials in the [book](https://icesat-2-2023.hackweek.io/tutorials/index.html)?

Reply:
Now :laughing: